<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraparamwidget</class>
 <widget class="QWidget" name="cameraparamwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QPushButton" name="bt_paramSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>参数配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_saveSet">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>保存配置</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1组</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2组</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3组</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4组</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>通道组</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_channel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>通道号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_config">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>单通道-高德</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双通道-高德</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>单通道-可见光</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双通道-可见光</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>motan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ivi</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>配置文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_setConfig">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_view_front">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>200</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8ae234;&quot;&gt;前视图&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../source.qrc">
     <normaloff>:/image/front.png</normaloff>:/image/front.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_view_right">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>250</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8ae234;&quot;&gt;右视图&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../source.qrc">
     <normaloff>:/image/right.png</normaloff>:/image/right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_view_top">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>300</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="俯视图">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8ae234;&quot;&gt;俯视图&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../source.qrc">
     <normaloff>:/image/top.png</normaloff>:/image/top.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../source.qrc"/>
 </resources>
 <connections/>
</ui>
